<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecDialog</class>
 <widget class="QDialog" name="SecDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="Add_Pushbutton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>130</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Add new Course</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_pushbutton">
   <property name="geometry">
    <rect>
     <x>735</x>
     <y>450</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>189</y>
     <width>661</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="loadData">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>20</y>
     <width>93</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
    </font>
   </property>
   <property name="text">
    <string>Load data</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>540</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="course">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="home">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="stu">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="course_plus">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="student_plus">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>back_pushbutton</zorder>
  <zorder>loadData</zorder>
  <zorder>label</zorder>
  <zorder>Add_Pushbutton</zorder>
  <zorder>tableView</zorder>
  <zorder>course</zorder>
  <zorder>home</zorder>
  <zorder>stu</zorder>
  <zorder>course_plus</zorder>
  <zorder>student_plus</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
